docs/user/starcraft_in_windows.md

```diff
@@ -32,8 +32,8 @@ AIClient or AIModule.
 - Download the game at https://battle.net/account/management/.
 - Run `setup.exe` and install the game in some folder.
   - `$STARCRAFT` will be from now on the installation directory.
-- Download the 1.16.1 patch at https://battle.net/support/en/article/classic-game-patches.
-  - Make sure you download the **Brood War** patch.
+- Download the 1.16.1 patch at http://ftp.blizzard.com/pub/broodwar/patches/PC/BW-1161.exe
+  - Make sure you download the **Brood War** patch 1.16.1.
```

Comment:
When Blizzard released 1.18 and announced Remastered (1.20) they removed the 1.15 installer that was available on their website. And the 1.16.1 patch doesn't work on either of the newer versions. So I think these instructions, sadly, no longer work.

If I recall correctly, though, the 1.18 MPQs are sufficient for OpenBW (even if the 1.18 executable doesn't itself support BWAPI). Does that sound correct @tscmoo ? If so, we could correct and simplify these instructions to, approximately, "go download 1.18"

Reply:
There's also the iccup version which works fine, so we could point to that.

Reply:
@ebetica @tscmoo confirms that the 1.18 MPQs should be sufficient for OpenBW. So we can just tell people to download that from Blizzard and skip the patching, albeit with the disclaimer that they can't use 1.18 directly like with ChaosLauncher.

Reply:
Linked to a Blizzard forum post on how to downgrade to 1.16.1